centreon/tests/e2e/support/commands.ts

```diff
@@ -228,12 +263,20 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add('updateCentreonConfig', () => {
```

Comment:
`updateCentreonConfig` does not explain the intention of the method
rename it to `enableDashboardFeature`